docs/docs-beta/docs/guides/metadata.md

```diff
@@ -5,9 +5,14 @@ sidebar_position: 40
 sidebar_label: 'Add metadata'
 ---
 
-Assets feature prominently in the Dagster UI. It's often helpful to attach information to assets to understand where they're stored, what they contain, and how they should be organized.
+Assets feature prominently in the Dagster UI. Attaching information to assets allows you to understand where they're stored, what they contain, and how they should be organized.
 
-In Dagster, you can attach ownership information, organize your assets with tags, attach rich, complex information with metadata, and link your assets with their source code.
+Using metadata in Dagster, you can:
+
+- Attach ownership information
+- Organize assets with tags
+- Attach rich, complex information
```

Comment:
Rich, complex information is vague

Reply:
An example would help here - maybe mentioning something about Markdown?